source/globals.d.ts

```diff
@@ -74,3 +74,11 @@ declare module 'react' {
 interface Node extends EventTarget {
 	cloneNode(deep?: boolean): this;
 }
+
+interface HTMLCollectionOf<T extends HTMLElement> {
+	[Symbol.iterator](): Iterator<T>;
+}
+
+interface NodeListOf<T extends ChildNode> {
+	[Symbol.iterator](): Iterator<T>;
+}
```

Comment:
I assume that the first one defaults `HTMLCollectionOf` to `HTMLElement`, which allows us to avoid having to specify `collection as HTMLElement[]` 👍 

But what's the second one for?

Reply:
It was for something I reverted, This was only needed if XO was updated.

Reply:
Let's undo this as well then, otherwise the reason it exists is lost